omnibus/files/server-ctl-cookbooks/infra-server/recipes/cleanup.rb

```diff
@@ -33,3 +33,17 @@
   recursive true
   action :delete
 end
+
+component_runit_service 'elasticsearch' do
+  action :disable
+end
+
+::Dir.glob(::File.join(node['private_chef']['user']['home'], '/elasticsearch/lib/log4j*')).each do |filename|
+  file filename do
+    action :delete
```

Comment:
add: 
```
notifies :restart, 'component_runit_service[elasticsearch]'
```
This will cause the restart to occur automatically after the files are removed in a way that  doesn't rely on the order of resources in the recipe.

Reply:
Updated it. Thank you!